server/src/main/java/com/broadcom/lsp/cobol/service/CobolTextDocumentService.java

```diff
@@ -251,21 +251,23 @@ private void clearAnalysedFutureObject(String uri) {
 
   private void analyzeDocumentFirstTime(String uri, String text, boolean userRequest) {
     registerDocument(uri, new CobolDocumentModel(text, AnalysisResult.empty()));
-    Future<?> docAnalysisFuture = threadPool.submit(
+    Future<?> docAnalysisFuture =
+        threadPool.submit(
             () -> {
               try {
                 CopybookProcessingMode copybookProcessingMode =
-                        CopybookProcessingMode.getCopybookProcessingMode(
-                                uri,
-                                userRequest
-                                        ? CopybookProcessingMode.ENABLED_VERBOSE
-                                        : CopybookProcessingMode.ENABLED);
+                    CopybookProcessingMode.getCopybookProcessingMode(
+                        uri,
+                        userRequest
+                            ? CopybookProcessingMode.ENABLED_VERBOSE
+                            : CopybookProcessingMode.ENABLED);
                 AnalysisResult result = engine.analyze(uri, text, copybookProcessingMode);
                 ofNullable(docs.get(uri)).ifPresent(doc -> doc.setAnalysisResult(result));
                 publishResult(uri, result, copybookProcessingMode);
                 outlineMap.get(uri).complete(result.getOutlineTree());
               } catch (Exception e) {
-                reportExceptionIfThrown(createDescriptiveErrorMessage("analysis", uri));
+                reportExceptionIfThrown(createDescriptiveErrorMessage("analysis", uri))
+                    .accept(null, e);
```

Comment:
Oh no =) It has become quite complicated. Could you please change it to:
```
LOG.error(createDescriptiveErrorMessage("analysis", uri), e);
```

Reply:
Done.